### ISSUE TITLE
Remove tooltip over compass when not activated

### ISSUE THREAD

**Issue by Lund-gren** [origo-map/origo#87]
You get a tooltip when holding the mouse over the compass even though it´s not activated, this sholud be removed.
![image](https://cloud.githubusercontent.com/assets/15904460/18905967/d701d448-8565-11e6-89dc-c2a83128a328.png)

**Comment by markus659** [comment]
It seems to be due to the fact that the .ol-rotate.ol-hidden class is not up-to-date/changed. I don't know if it's the whole css style for OL that needs to updated for Origo or if it's specific for this class.

This is what the Origo css looks like:

```css
.ol-rotate.ol-hidden {
  opacity: 0;
}
```

And in OL:

```css
.ol-rotate.ol-hidden {
  opacity: 0;
  visibility: hidden;
  transition: opacity .25s linear, visibility 0s linear .25s;
}
```

It is the visibilty option that also seems to hide the tooltip. You can try it with this example: http://codepen.io/anon/pen/gwRxpk

If the visibility property is removed, the tooltip will show.

![rotate](https://cloud.githubusercontent.com/assets/15904932/18907338/925f19e4-856b-11e6-894c-ac64ba4bf0a2.jpg)

**Comment by afogelberg** [comment]
You're right @markus659  . The css from ol needs to be updated.

**Comment by afogelberg** [comment]
Fixed with 92b03a7961fdb1cc4e224eb9b0c475f1930db2cf and 1700417b1bb80aada28a730ecda1fa075b2d77bb
